### ISSUE TITLE
icon interface doesn't accept icons with path not in ~/.config/LightweightQtDRMStreamViewer/providers/

### ISSUE THREAD

**Issue by omni6** [magiruuvelvet/LightweightQtDRMStreamViewer#3]
like title

when set path outside standard directory text disapears and no icon is shown.

btw, you're on arch!? mentioned periodically dropouts, if app get started you need `--disable-seccomp-filter-sandbox` argument.

**Comment by magiruuvelvet** [comment]
this is indented behavior, i may improve this when refactoring the application

also what do you mean with the second statement about `--disable-seccomp-filter-sandbox`?

**Comment by omni6** [comment]
statting without can result in connection losses and dropouts:

```
../../../../qtwebengine-everywhere-src-5.10.0/src/3rdparty/chromium/sandbox/linux/seccomp-bpf-helpers/sigsys_handlers.cc:**CRASHING**:seccomp-bpf failure in syscall 0025
Received signal 11 SEGV_MAPERR 000000000019
```

starting with `--disable-seccomp-filter-sandbox` is the workaround.

**Comment by magiruuvelvet** [comment]
Weird. Arch doesn't seem to have this problem. Sounds like an upstream issue to me.

**Comment by omni6** [comment]
i am on arch and i had it :) , but you're right it is a strange issue and and seems to be random.

**Comment by magiruuvelvet** [comment]
I added a hint to the README in case other users run into the same problem.

**Comment by magiruuvelvet** [comment]
regarding your other issue. i opened a new issue to not forget about this
#11
as said only relative paths are supported right now. you could always add `..` to go back one directory, but that's just a mess

**Comment by magiruuvelvet** [comment]
@omni6 implemented in 32e11eaff54f4465eec64ed192b21f010d4a7485, icons can now be placed outside that directory
